src/AbstractOptionsAwareDataTableType.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+
```

Comment:
Did you run `vendor/bin/php-cs-fixer fix` before committing? I am surprised that it would leave two blank lines.

Reply:
Nope. I did not know that bundle. I'll take care of it 👍